code.js

```diff
@@ -0,0 +1,43 @@
+//Settings
+const minSpeed = 0.5;
+const maxSpeed = 3;
+let isDragStarted = false;
+
+const speedControll = document.querySelector('.speedcontrol');
+const speedControllFilled = document.querySelector('.speedcontrol_filled');
+const video = document.querySelector('video');
+const speedvalue = document.querySelector('.speedvalue');
+
+//Listeners
+speedControll.addEventListener('mousedown', (e) => {
+    isDragStarted = true;
+    renderSpeedControl(e);
+});
+speedControll.addEventListener('mouseup', () => {
+    isDragStarted = false;
+});
+speedControll.addEventListener('mouseout', () => {
+    isDragStarted = false;
+});
+speedControll.addEventListener('mousemove', (e) => {
+    if(isDragStarted) renderSpeedControl(e);
+})
+
+//Functions
+function renderSpeedControl (e) {
+    console.log(e);
+    let percent = (e.offsetY / 700);
```

Comment:
What does mean - 700? if you use constant value - please create  external variable for that

Reply:
done